cinch/roles/jenkins_slave/templates/sysconfig_jenkins_swarm

```diff
@@ -6,4 +10,4 @@ SWARM_SLAVE_LABEL="{{ jslave_label }}"
 SWARM_ROOT="{{ jswarm_home }}"
 SWARM_USERNAME="{{ (jenkins_slave_username == '') | ternary('', '-username') }} {{ jenkins_slave_username }}"
 SWARM_PASSWORD="{{ (jenkins_slave_password == '') | ternary('', '-password') }} {{ jenkins_slave_password }}"
-SWARM_EXTRA_ARGS="-mode exclusive -disableSslVerification -deleteExistingClients"
+SWARM_EXTRA_ARGS="-mode exclusive -disableSslVerification -deleteExistingClients {{ jswarm_extra_args }}"
```

Comment:
The main reason I originally moved away from the "construct the full command in Ansible" method when we moved to systemd/upstart was because doing so left no trace on the filesystem of an easily editable configuration file. Someone who was tweaking an on-disk running slave would need to edit the systemd (or upstart or shell) file directly instead of being able to edit a configuration file.

Maintainability of this software argues for moving back to constructing the string in Ansible, but there are other technical difficulties there. Such as
* How do we properly account for optional -password and -username args in a Docker container, where we don't know if they're supplied until runtime?
* How do we make it easy on a user to edit a configuration file to test changes to their system in real time?